var/spack/repos/builtin/packages/py-bluepyemodel/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBluePyEModel(PythonPackage):
+    """Python library to optimize and evaluate electrical models."""
+
+    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/cells/BluePyEModel"
+    git      = "ssh://bbpcode.epfl.ch/cells/BluePyEModel"
+
+    version('0.0.3', tag='bluepyemodel-v0.0.3')
+
+    depends_on('py-setuptools', type='build')
+
+    depends_on('py-click@7.0:', type='run')
+    depends_on('py-numpy@1.15.0:', type='run')
+    depends_on('py-scipy@0.13:', type='run')
+    depends_on('py-h5py@2.9:', type='run')
+    depends_on('py-matplotlib@2.2:', type='run')
+    depends_on('py-pandas@0.24:', type='run')
+    depends_on('py-efel@3.0.80:', type='run')
+    depends_on('py-morph-tool@2.4.4:', type='run')
+    depends_on('bluepyopt @ git+http://github.com/BlueBrain/BluePyOpt@CMA_clean#egg=bluepyopt', type='run')
+    depends_on('bluepyefe @ git+http://github.com/BlueBrain/BluePyEfe@BPE2#egg=bluepyefe', type='run')
```

Comment:
AFAIK you can only add dependencies of other spack packages.
This means that you should create a specific version in spack for `bluepyopt` and `bluepyefe`, and add these dependencies here.
It would be simpler if it's possible to merge the changes in those packages to the main branch, but I guess it's not possible?

Reply:
I see, thanks! @wvangeit @DrTaDa what would you think would be best here? Create 2 spack packages for these 2 branches? Or we make BluePyEmodel depend on the spack versions by default, and only use these branches in dev in local for now?

Reply:
I think we have to go for the first option, as BPEM will not work without BPE2 and CMA.

Reply:
Do you know how to do that @DrTaDa , and which commit to ping if we can?

Reply:
For the commits, we can take the last one for both BPE2 and CMA_clean.
I guess we have to create two new packages in spake and give them different name than the original ones ?